OpenRA.Mods.Common/Widgets/ControlGroupsWidget.cs

```diff
@@ -40,7 +40,7 @@ public class ControlGroupsWidget : Widget
 		public readonly string JumpToGroupKeyPrefix = null;
 
 		[CustomLintableHotkeyNames]
-		public static IEnumerable<string> LinterHotkeyNames(MiniYamlNode widgetNode, Action<string> emitError, Action<string> emitWarning)
+		public static IEnumerable<string> LinterHotkeyNames(MiniYamlNode widgetNode, Action<string> emitError)
```

Comment:
`CustomLintableHotkeyNames` appears to involve reflection so I imagine this would break as-is.

Options:
- Restore the parameter and suppress it
- It seems we've removed `emitWarning` from all sites with this attribute, maybe we should just remove `emitWarning` support from `CheckChromeHotkeys.cs` as not required. We would then keep the parameter removals.